atlp-rwanda/ecommerce-app-legends-bn
# feature for auto actions

#### What does this PR do?
- To prompt a user to update his or her password for securty reason
#### Description of Task to be completed?

- allow user too be prompted to update password every year
- sechedule cron jobs to run every day at 17:00
- Routes:
- `GET: /api/v1/users/password/update`


#### How should this be manually tested?

- git clone from repo
- switch into branch
- run npm i
- run npm run dev
- check your email daily
- to test app notification:
- add this html file into src folder for testing app notfications
-  [index.html.zip](https://github.com/atlp-rwanda/ecommerce-app-legends-bn/files/11155703/index.html.zip)
- with postman run the route above
- wits swagger doc search http://localhost:{port}/docs/#/Auth/put_api_v1_users_password_update

###  screenshots

<img width="1440" alt="Screen Shot 2023-04-05 at 08 25 32" src="https://user-images.githubusercontent.com/99626447/229998551-875b0911-8133-47ed-8ae7-81053f3b98c1.png">
<img width="1063" alt="Screen Shot 2023-04-05 at 08 33 50" src="https://user-images.githubusercontent.com/99626447/229999814-65d3739d-16b1-4234-81e2-d751d56d0230.png">
<img width="1440" alt="Screen Shot 2023-04-05 at 08 35 01" src="https://user-images.githubusercontent.com/99626447/229999998-e7307d80-67b8-4357-a65b-9fec3a1a1faa.png">
<img width="1280" alt="Screen Shot 2023-04-05 at 08 37 20" src="https://user-images.githubusercontent.com/99626447/230000477-51a15a2a-4eef-4307-a11a-a2e12de5e870.png">
